integration-cli/docker_cli_network_unix_test.go

```diff
@@ -1012,6 +1012,10 @@ func (s *DockerNetworkSuite) TestDockerNetworkHostModeUngracefulDaemonRestart(c
 		cName := fmt.Sprintf("hostc-%d", i)
 		out, err := s.d.Cmd("run", "-d", "--name", cName, "--net=host", "--restart=always", "busybox", "top")
 		c.Assert(err, checker.IsNil, check.Commentf(out))
+
+		// verfiy container has finished starting before killing daemon
+		err = s.d.waitRun(fmt.Sprintf("hostc-%d", i))
```

Comment:
oh wait, it already is. Can we reuse `cName` from L#1012, or am I missing something?